tests/core/test_dataset_filters.py

```diff
@@ -285,6 +303,24 @@ def test_slice_filter(datasets):
     assert result.n_points < 1
 
 
+def test_slice_filter_plane(ant):
+    origin = (1, 2, 3)
+    normal = (4, 5, 6)
+    plane = pv.Plane(center=origin, direction=normal)
+    output_no_plane = ant.slice(origin=origin, normal=normal)
+    output_with_plane = ant.slice(plane=plane)
+    assert np.allclose(output_no_plane.bounds, output_with_plane.bounds)
```

Comment:
This test is exactly the same for the four filters which have this feature. I can parametrize it instead into a single test if that's preferred.